ui/pages/unlock-page/unlock-page.component.js

```diff
@@ -246,96 +239,78 @@ export default class UnlockPage extends Component {
             width={BlockSize.Full}
             alignItems={AlignItems.center}
           >
-            <Box
-              marginTop={6}
-              marginBottom={isBeta() || isFlask() ? 6 : 0}
-              className="unlock-page__mascot-container"
-            >
+            <Box className="unlock-page__mascot-container" marginBottom={4}>
               {this.renderMascot()}
               {isBeta() ? (
-                <Box className="unlock-page__mascot-container__beta">
+                <Text
+                  className="unlock-page__mascot-container__beta"
+                  backgroundColor={BackgroundColor.primaryDefault}
+                  color={TextColor.primaryInverse}
+                  padding={1}
+                  borderRadius={BorderRadius.LG}
+                  textTransform={TextTransform.Uppercase}
+                  fontWeight={FontWeight.Medium}
+                >
                   {t('beta')}
-                </Box>
+                </Text>
               ) : null}
             </Box>
             <Text
               data-testid="unlock-page-title"
               as="h1"
-              variant={TextVariant.headingLg}
-              marginTop={1}
-              marginBottom={2}
+              variant={TextVariant.displayMd}
+              marginBottom={6}
+              fontWeight={FontWeight.Medium}
               color={TextColor.textDefault}
+              textAlign={TextAlign.Center}
             >
               {t('welcomeBack')}
             </Text>
             <FormTextField
               id="password"
-              label={
-                <Box
-                  display={Display.Flex}
-                  width={BlockSize.Full}
-                  justifyContent={JustifyContent.spaceBetween}
-                  alignItems={AlignItems.center}
-                  marginBottom={1}
-                >
-                  <Text variant={TextVariant.bodyMdMedium}>
-                    {t('password')}
-                  </Text>
-                </Box>
-              }
```

Comment:
Removing the label, to ensure accessibility still remains intact we are adding an aria-label to the input on line [275](https://github.com/MetaMask/metamask-extension/pull/32805/files#diff-f6f23f7c3cf523f0e66d1892bfd99b9179bae4978e855c4b6e683d5d0b21981dR275)